config.env.py

```diff
@@ -33,4 +32,13 @@
 MAIL_SERVER = env.get("MAIL_SERVER", "thoth.csh.rit.edu")
 MAIL_USERNAME = env.get("MAIL_USERNAME", "rideboard@csh.rit.edu")
 MAIL_PASSWORD = env.get("MAIL_PASSWORD", None)
-MAIL_USE_TLS = True
\ No newline at end of file
+MAIL_USE_TLS = True
+
+
+
+
+
+
+
+
+
```

Comment:
Trailing newlines! Bad!

Reply:
https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/221cdfdf9f3b8de712df4e2b1f94cb7cf2ee44ad

Reply:
That's a lot of trailing newlines :blob-sweats: